lumicks/pylake/kymo.py

```diff
@@ -249,6 +249,56 @@ def plot_rgb(self, **kwargs):
         image = image / np.max(image)
         return self._plot(image, **kwargs)
 
+    def _downsample_channel(self, n, xy, reduce=np.mean, where="center"):
+        force = self.file._get_force(n, xy)
+        # downsample exactly over the scanline time range
+        min_times = self.timestamps[0, :].astype(np.int64)
+        max_times = self.timestamps[-1, :].astype(np.int64)
+        time_ranges = [(mini, maxi) for mini, maxi in zip(min_times, max_times)]
+        return force.downsampled_over(time_ranges, reduce=reduce, where=where)
+
+    def plot_with_force(self, force_channel, color_channel, aspect_ratio=0.25,
+                        reduce=np.mean, where="center", **kwargs):
+        """Plot kymo with force channel downsampled over scan lines
+
+        Parameters
+        ----------
+        force_channel: str
+            name of force channel to downsample and plot
+        color_channel: str
+            color channel of kymo to plot ('red', 'green', 'blue', 'rgb')
+        aspect_ratio: float
+            axes aspect ratio
```

Comment:
I'm not an avid user of `matplotlib`, so I could not make sense of what a single value `aspect_ratio` meant.

I suggest you add something in the lines of "Set the aspect of the axis scaling, i.e. the ratio of y-unit to x-unit", which I shamelessly copied from https://matplotlib.org/3.1.1/api/_as_gen/matplotlib.axes.Axes.set_aspect.html  : D